codalab_service.py

```diff
@@ -173,7 +173,7 @@ def has_callable_default(self):
     CodalabArg(
         name='bundle_mount',
         help='Path to bundle data (just for mounting into Docker)',
-        default=var_path('bundles'),
+        default=var_path('/tmp'),
```

Comment:
should just be `default='/tmp'`

Reply:
I was not exactly sure here because `var_path` function seems to return a lambda function. Is it OK to just set it to a string?  `var_path` calls `os.path.join` function which will correctly handle appending a absolute path. Maybe `default=lambda _: '/tmp'` for more clearance?

Reply:
`/tmp` is fine (lambda or constant works).